docs/resources/security_monitoring_rule.md

```diff
@@ -54,19 +54,20 @@ resource "datadog_security_monitoring_rule" "myrule" {
 
 ### Required
 
-- `case` (Block List, Min: 1, Max: 10) Cases for generating signals. (see [below for nested schema](#nestedblock--case))
 - `message` (String) Message for generated signals.
 - `name` (String) The name of the rule.
 
 ### Optional
 
+- `case` (Block List, Max: 10) Cases for generating signals. (see [below for nested schema](#nestedblock--case))
 - `enabled` (Boolean) Whether the rule is enabled. Defaults to `true`.
 - `filter` (Block List) Additional queries to filter matched events before they are processed. (see [below for nested schema](#nestedblock--filter))
 - `has_extended_title` (Boolean) Whether the notifications include the triggering group-by values in their title. Defaults to `false`.
 - `options` (Block List, Max: 1) Options on rules. (see [below for nested schema](#nestedblock--options))
 - `query` (Block List) Queries for selecting logs which are part of the rule. (see [below for nested schema](#nestedblock--query))
 - `signal_query` (Block List) Queries for selecting logs which are part of the rule. (see [below for nested schema](#nestedblock--signal_query))
 - `tags` (Set of String) Tags for generated signals.
+- `third_party_case` (Block List, Max: 10) Cases for generating signals for third party rules. Only required and accepted for third party rules (see [below for nested schema](#nestedblock--third_party_case))
```

Comment:
```suggestion
- `third_party_case` (Block List, Max: 10) Cases for generating signals for third-party rules. Only required and accepted for third-party rules (see [below for nested schema](#nestedblock--third_party_case))
```

Reply:
Thanks! Applied in https://github.com/DataDog/terraform-provider-datadog/pull/2231/commits/1e4872ae66b9b61487da7cfbdc524101cda4f968.